llama-index-integrations/embeddings/llama-index-embeddings-huggingface/llama_index/embeddings/huggingface/base.py

```diff
@@ -32,6 +32,17 @@
 
 
 class HuggingFaceEmbedding(BaseEmbedding):
+    """
+    Args:
+        parallel_process (bool): Default to False. If True it will start a multi-process pool to process the encoding
+            with several independent processes.
+
+        target_devices (List[str], optional): It will only taken into account if `parallel_process` = `True`. PyTorch
+            target devices, e.g. ["cuda:0", "cuda:1", ...], ["npu:0", "npu:1", ...], or ["cpu", "cpu", "cpu", "cpu"].
+            If target_devices is None and CUDA/NPU is available, then all available CUDA/NPU devices will be used.
+            If target_devices is None and CUDA/NPU is not available, then 4 CPU devices will be used.
```

Comment:
thanks for the class docstring. We should probably expand this to all fields and methods?